dev/sg/sg_setup_mac.go

```diff
@@ -0,0 +1,263 @@
+			},
+		},
+		autoFixing: false,
+	},
+	{
+		name: "Install base utilities (git, docker, ...)",
+		dependencies: []*dependency{
+			{name: "git", check: getCheck("git"), instructionsCommands: `brew install git`},
+			{name: "gnu-sed", check: check.InPath("gsed"), instructionsCommands: "brew install gnu-sed"},
+			{name: "comby", check: check.InPath("comby"), instructionsCommands: "brew install comby"},
+			{name: "pcre", check: check.InPath("pcregrep"), instructionsCommands: `brew install pcre`},
+			{name: "sqlite", check: check.InPath("sqlite3"), instructionsCommands: `brew install sqlite`},
+			{name: "jq", check: check.InPath("jq"), instructionsCommands: `brew install jq`},
+			{name: "bash", check: check.CommandOutputContains("bash --version", "version 5"), instructionsCommands: `brew install bash`},
+			{
+				name: "rosetta",
+				check: check.Any(
+					check.CommandOutputContains("uname -m", "x86_64"), // will return true on non-m1 macs
+					check.CommandExitCode("pgrep oahd", 0)),           // oahd is the process running rosetta
+				instructionsCommands: `softwareupdate --install-rosetta --agree-to-license`,
+			},
+			{
+				name:                 "docker",
+				check:                getCheck("docker-installed"),
+				instructionsCommands: `brew install --cask docker`,
+			},
+		},
+		autoFixing: true,
+	},
+	{
+		name: "Clone repositories",
+		dependencies: []*dependency{
+			{
+				name:  "SSH authentication with GitHub.com",
+				check: check.CommandOutputContains("ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -T git@github.com", "successfully authenticated"),
+				instructionsComment: `` +
+					`Make sure that you can clone git repositories from GitHub via SSH.
+See here on how to set that up:
+
+https://docs.github.com/en/authentication/connecting-to-github-with-ssh
+`,
+			},
+			{
+				name:                 "github.com/sourcegraph/sourcegraph",
+				check:                checkInMainRepoOrRepoInDirectory,
+				instructionsCommands: `git clone git@github.com:sourcegraph/sourcegraph.git`,
+				instructionsComment: `` +
+					`The 'sourcegraph' repository contains the Sourcegraph codebase and everything to run Sourcegraph locally.`,
+			},
+			{
+				name:                 "github.com/sourcegraph/dev-private",
+				check:                checkDevPrivateInParentOrInCurrentDirectory,
+				instructionsCommands: `git clone git@github.com:sourcegraph/dev-private.git`,
+				instructionsComment: `` +
+					`In order to run the local development environment as a Sourcegraph employee,
+you'll need to clone another repository: github.com/sourcegraph/dev-private.
+
+It contains convenient preconfigured settings and code host connections.
+
+It needs to be cloned into the same folder as sourcegraph/sourcegraph,
+so they sit alongside each other, like this:
+
+   /dir
+   |-- dev-private
+   +-- sourcegraph
+
+NOTE: You can ignore this if you're not a Sourcegraph employee.
+`,
+				onlyEmployees: true,
```

Comment:
```suggestion
				onlyTeammates: true,
```

IIRC there was a handbook page saying we legally cannot use the word "employee". For example, I'm technically not an employee, so we use "teammates"

Reply:
We should update docstrings here too